core/perf_test/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if(NOT Kokkos_ENABLE_BENCHMARKS)
 endif()
 
 # Find or download google/benchmark library
-find_package(benchmark QUIET 1.5.6)
+find_package(benchmark QUIET 1.9.1)
```

Comment:
Why are bumping our minimum requirements (versus just updating the fallback version that we pull)?

Reply:
ping

Reply:
We want to depend on the package following 1.5.6 (1.6 or newer) since 1.5.6 has deprecations in the header that error out. For consistency we could depend on the same that we pull.